schema/examples/direct/manifest.xml

```diff
@@ -2,11 +2,14 @@
 <direct-signature-job-manifest xmlns="http://signering.posten.no/schema/v1">
     <signer>
         <personal-identification-number>12345678910</personal-identification-number>
+        <signing-ceremony>
+            <method>ELECTRONIC_SIGNATURE</method>
+        </signing-ceremony>
     </signer>
     <sender>
         <organization-number>123456789</organization-number>
     </sender>
-    <document href="document.pdf" mime="application/pdf">
+    <document href="document.pdf" mime="application/pdf" authentication-level="3">
```

Comment:
Næh, jeg tror kanskje rasjonalet var at det er _metadata_ knyttet til dokumentet (på samme måte som filnavn og filtype) eller enn verdier som blir presentert til brukeren (tittel og melding). Føltes hensiktsmessig å gruppere sammen filnavn og -type. Men det er ikke så viktig for meg.